.github/workflows/maven.yml

```
@@ -32,3 +32,22 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Build
         run: mvn ${{ env.maven_commands }}
+  deploy:
+    if: contains('refs/heads/develop', github.ref)
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

```suggestion
          java-version: 8
```

Due to fundamental compatibility issue in the Java ecosystem, artifacts should always be produced using the minimal Java version.

---

I am okay with that but previously in travis we had ``openjdk11``

---

I have pushed a commit since the suggestion is incorrect. it should ``1.8``

---

Agreed. That was probably done before we realized compiling with higher versions of Java could cause issues. See https://forum.image.sc/t/fiji-wont-open-czi-files-after-update-there-was-a-problem-with-the-class-java/40932/8 for a recent occurence i can think of.